package.json

```diff
@@ -47,11 +47,19 @@
     "@reduxjs/toolkit": "^1.8.0",
     "@turf/turf": "^6.5.0",
     "d3": "^7.0.0",
+    "d3-axis": "^3.0.0",
+    "d3-brush": "^3.0.0",
+    "d3-path": "^3.0.1",
+    "d3-scale": "^4.0.2",
+    "d3-selection": "^3.0.0",
+    "d3-time-format": "^4.1.0",
+    "geojson-world-map": "^0.0.2",
     "mapbox-gl": "npm:empty-npm-package@^1.0.0",
     "maplibre-gl": "^2.1.9",
     "next": "^12.1.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-leaflet": "^3.2.5",
```

Comment:
the Map page uses `react-map-gl` - would be cool if we could use it for the visual query page as well